gulpfile.js

```diff
@@ -85,7 +85,7 @@ gulp.task('vendor', function vendorTask(done) {
 
 // based on https://github.com/gulpjs/gulp/blob/master/docs/recipes/browserify-with-globs.md,
 // except we use the Promise returned by globby, instead of passing it a callback
-gulp.task('scripts', ['lint'], function scriptsTask() {
+gulp.task('scripts', function scriptsTask() {
```

Comment:
Hmm, could switch this on an env var so I can have the constant pestering I actually like